Directory.Build.props

```
@@ -46,6 +46,10 @@
 	<PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
 		<ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
 	</PropertyGroup>
+    
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0' or '$(TargetFramework)' == 'net9.0'">
```

```suggestion
  <PropertyGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
```

This is a more maintainable way of writing this condition.

---

Thanks, this is now removed

---

@dualtagh we probably have similar issues in all our repos. Can you either fix them all or open issues? thank you.